tds/modules/code/model.py

```diff
@@ -1,23 +1,43 @@
 """
 TDS Code Data Model Definition.
 """
-from typing import Optional
+from typing import Dict, List, Optional
+
+from pydantic import BaseModel, Field
 
 from tds.db.base import TdsModel
-from tds.db.enums import ProgrammingLanguage
+
+
+class CodeFile(BaseModel):
+    """
+    CodeFile Data Model for individual file entries in the main Code model.
+    """
+
+    language: Optional[str] = Field(description="Programming language of the file.")
+    dynamics: Optional[str] = Field(
+        None,
+        description="String indicating the line numbers in the file that contain the dynamics, e.g., 'L205-L213'.",
+    )
```

Comment:
We were talking about having a description/name field for that dynamic to help the user remember which is which, etc. Displaying them on a list like symbols on GitHub or adding them in Notes in the futur.

Reply:
@YohannParis just to make sure I got it right--optional `name` and `description` for the `dynamics`, not the file? That is no problem to add.

Reply:
Addressed in latest commit